router-tests/kafka_events_test.go

```diff
@@ -116,7 +116,7 @@ func TestKafkaEvents(t *testing.T) {
 
 			go func() {
 				wg.Wait()
-				require.NoError(t, client.Close())
```

Comment:
Produced "use of closed connection" issues which can be ignored because in some conditions the connection has been already closed which is fine.